test/cluster/test_maintenance_mode.py

```diff
@@ -90,7 +90,7 @@ async def test_maintenance_mode(manager: ManagerClient):
         await manager.server_stop_gracefully(server_a.server_id)
```

Comment:
> Setting the value of `maintenance_mode` to the string `"false"` does NOT disable maintenance mode.

It does. The issue is that the testing framework incorrectly handles the case where `in_maintenance_mode()` returns `"false"`.